spark-testing/spark_utils.py

```diff
@@ -179,7 +187,7 @@ def submit_job(
 
 def check_job_output(task_id, expected_output):
     LOGGER.info('Waiting for task id={} to complete'.format(task_id))
-    shakedown.wait_for_task_completion(task_id, timeout_sec=JOB_WAIT_TIMEOUT_SEC)
+    shakedown.wait_for_task_completion(task_id, timeout_sec=JOB_WAIT_TIMEOUT_SECONDS)
```

Comment:
AFAIK, SDK 0.55.x got rid of shakedown library. This may be separate task but we should check if we are using the shakedown in spark tests and create tickets to remove them eventually. You can double check with @takirala  and @kvish about this.

Reply:
@imaxxs, we should depend on SDK in a different way (not by duplicating the code). Currently, codebases diverged and moving off of shakedown might be painful. Anyway, we have to do it.